licenses/management/commands/publish.py

```diff
@@ -0,0 +1,154 @@
+import subprocess
+from argparse import ArgumentParser
+
+from django.core.management import BaseCommand
+
+from cc_licenses.settings.base import TRANSLATION_REPOSITORY_DIRECTORY
+
+
+def run_django_distill():
+    """Outputs static files into the specified directory determined by settings.base.DISTILL_DIR
+
+    The input option that subprocess provides is important. If a build dir does not exist we must
+    say yes the first time we run this command in order to create the directory. Once the build dir
+    is created the force argument will auto-create the build without our input. This is way we are
+    using subprocess in this method over management.call_command.
+    """
+    cmd = ["python", "manage.py", "distill-local", "--quiet", "--force"]
+    return subprocess.run(cmd, text=True, input="YES",)
```

Comment:
As we discussed just now on our call, instead of calling the management command to get django-distill to run, we can just make the couple of calls that the django-distill distill-local handler does.

Reply:
Making the changes. I don't understand the benefit of the change though.